examples/compute/flavor/min_disk/main.tf

```diff
@@ -0,0 +1,15 @@
+# If the exact amount of disk is not so important to you, then you can specify the minimum value that will satisfy you
+# and flavor with minimum of disk will be automatically selected for you.
+data "vkcs_compute_flavor" "basic1" {
+  vcpus    = 4
+  ram      = 8192
+  min_disk = 30
```

Comment:
we do not want examples on min_disk. lets move this to an acceptance test

Reply:
It's already exist as acceptance test, just removed example